services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/grpc/GrpcCommonSpec.groovy

```diff
@@ -0,0 +1,34 @@
+package org.openkilda.functionaltests.spec.grpc
+
+import org.openkilda.functionaltests.BaseSpecification
+
+import groovy.util.logging.Slf4j
+import spock.lang.Shared
+
+@Slf4j
+class GrpcCommonSpec extends BaseSpecification {
+    @Shared
+    String switchIp
+
+    def setupOnce() {
+        requireProfiles("hardware")
+        def nFlowSwitch = northbound.activeSwitches.find { it.description =~ /NW[0-9]+.[0-9].[0-9]/ }
+        def pattern = /(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\-){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)/
+        switchIp = (nFlowSwitch.address =~ pattern)[0].replaceAll("-", ".")
```

Comment:
These four lines are used 3 times in the next specs. Need to avoid duplication.

Reply:
Fixed